### DISCUSSION TITLE
RAG API debug error with version

### DISCUSSION THREAD

**Discussion by ervet** [danny-avila/LibreChat#4658] [General]
Hello,

upgraded to the newest Librechat version and now it  gives me with lite version of the RAG API Image the following Error. It worked before.

EMBEDDINGS_PROVIDER, EMBEDDINGS_MODEL is defined. I sadly do not know what he is expecting with the Version and where to put the version number. 

Any hints ?

Greetings
Erik  

rag_api-1         | Traceback (most recent call last):
rag_api-1         |   File "/app/main.py", line 46, in <module>
rag_api-1         |     from psql import PSQLDatabase, ensure_custom_id_index_on_embedding, pg_health_check
rag_api-1         |   File "/app/psql.py", line 3, in <module>
rag_api-1         |     from config import DSN, logger
rag_api-1         |   File "/app/config.py", line 252, in <module>
rag_api-1         |     embeddings = init_embeddings(EMBEDDINGS_PROVIDER, EMBEDDINGS_MODEL)
rag_api-1         |   File "/app/config.py", line 189, in init_embeddings
rag_api-1         |     return AzureOpenAIEmbeddings(
rag_api-1         |   File "/usr/local/lib/python3.10/site-packages/pydantic/main.py", line 212, in __init__
rag_api-1         |     validated_self = self.__pydantic_validator__.validate_python(data, self_instance=self)
rag_api-1         | pydantic_core._pydantic_core.ValidationError: 1 validation error for AzureOpenAIEmbeddings
rag_api-1         |   Value error, Must provide either the `api_version` argument or the `OPENAI_API_VERSION` environment variable [type=value_error, input_value={'azure_deployment': 'tex...one, 'model_kwargs': {}}, input_type=dict]
rag_api-1         |     For further information visit https://errors.pydantic.dev/2.9/v/value_error

**Comment by danny-avila** [comment]
Are you using Azure for embeddings?

**Reply by ervet** [reply]
I still have this problem - an still can't figured out why this is happing .. in the meantime upgraded to 0,7.6 but the error still exists. I don't know what I have done wrong.

**Comment by ervet** [comment]
Yes I do ...  I have set the following 

RAG_AZURE_OPENAI_ENDPOINT=https://$%$%$.openai.azure.com
RAG_AZURE_OPENAI_API_KEY=$%$%$
EMBEDDINGS_PROVIDER=azure
EMBEDDINGS_MODEL=text-embedding-ada-002
DEBUG_RAG_API=true
#RAG_USE_FULL_CONTEXT=true
#PDF_EXTRACT_IMAGES=true

**Comment by ervet** [comment]
🙈 Maybe my bad. I know set the the Version via 

RAG_AZURE_OPENAI_API_VERSION

I don't know why do not have to set in my other installations. 

Greetings
Erik

**Comment by ervet** [comment]
closed - RAG_AZURE_OPENAI_API_VERSION needs to be set
